src/components/NewOversiktTable.tsx

```diff
@@ -0,0 +1,153 @@
+import React, { ReactElement } from 'react';
+import { Checkbox, SortState, Table } from '@navikt/ds-react';
+import { VeilederColumn } from '@/components/VeilederColumn';
+import { PersonData } from '@/api/types/personregisterTypes';
+import {
+  lenkeTilModia,
+  lenkeTilModiaEnkeltperson,
+  lenkeTilModiaEnkeltpersonFnr,
+} from '@/utils/lenkeUtil';
+import { useAktivBruker } from '@/data/modiacontext/useAktivBruker';
+import { PersonRadVirksomhetColumn } from '@/components/PersonRadVirksomhetColumn';
+import { OppfolgingstilfelleDTO } from '@/api/types/personoversiktTypes';
+import { FristColumn } from '@/components/FristColumn';
+import { Labels } from '@/components/Labels';
+import { SortingKey, SortingType, useSorting } from '@/hooks/useSorting';
+
+const getVarighetOppfolgingstilfelle = (
+  oppfolgingstilfelle: OppfolgingstilfelleDTO | undefined
+): string => {
+  return oppfolgingstilfelle
+    ? `${oppfolgingstilfelle.varighetUker} uker`
+    : 'Ukjent';
+};
+
+interface Props {
+  personListe: [string, PersonData][];
+  selectedRows: string[];
+  setSelectedRows: (rows: string[]) => void;
+  sortingType: SortingType;
+  setSortingType: (sortingType: SortingType) => void;
+}
+
+export const NewOversiktTable = ({
+  personListe,
+  selectedRows,
+  setSelectedRows,
+  sortingType,
+  setSortingType,
+}: Props): ReactElement => {
+  const aktivBruker = useAktivBruker();
+  const { columns, toSortState, toSortingType } = useSorting();
+
+  const sortState: SortState | undefined = toSortState(sortingType);
+  const handleSort = (sortKey: string | undefined) => {
+    if (sortKey && sortState) {
+      setSortingType(toSortingType(sortKey as SortingKey, sortState.direction));
+    } else {
+      setSortingType('NONE');
+    }
+  };
+
+  const onPersonClick = (fnr: string, personData: PersonData) => {
+    aktivBruker.mutate(fnr, {
+      onSuccess: () => {
+        window.location.href = lenkeTilModia(personData);
+      },
+    });
+  };
+
+  const allRowsSelected = personListe.length === selectedRows.length;
+  const anyRowsSelected = selectedRows.length > 0;
+  const isRowSelected = (fnr: string) => selectedRows.includes(fnr);
+
+  const toggleSelectedRow = (fnr: string) => {
+    if (isRowSelected(fnr)) {
+      setSelectedRows(selectedRows.filter((value) => value !== fnr));
+    } else {
+      setSelectedRows([...selectedRows, fnr]);
+    }
+  };
+
+  const toggleSelectAllRows = () => {
+    if (anyRowsSelected) {
+      setSelectedRows([]);
+    } else {
+      setSelectedRows(personListe.map(([fnr]) => fnr));
+    }
+  };
+
+  return (
+    <Table
+      sort={sortState}
```

Comment:
Aksel Table bruker egen `SortState`-type for sortering (https://aksel.nav.no/komponenter/core/table#tabledemo-sortable)
Må derfor ha noe kode i `useSorting` for å mappe mellom `SortState` og `SortingType` som brukes av `<Sorteringsrad>` (gammel tabell). Når vi fjerner gammel tabell kan vi også skrive om til å bare bruke `SortState`.